src/machine/eval.rs

```diff
@@ -117,6 +115,10 @@ impl Machine {
         // return the body
         (true, body)
     }
+    /// Evaluation step for application of primitive functions (prim arg)
+    fn step_primitive_app(&mut self, _prim: Primitive, _arg: Box<Term>) -> (bool, Term) {
+        todo!()
```

Comment:
Just for completeness sake can you add `"There aren't any primitives yet!"` as argument to `todo!`?